diff --git a/std/tls13_session.no b/std/tls13_session.no
index 477961a..10dff7f 100644
--- a/std/tls13_session.no
+++ b/std/tls13_session.no
@@ -205,6 +205,8 @@ funksjon _tenar_motta(s: ordbok, records: liste) -> ordbok {
         la th_sfin = ks.transcript_hash(s["transcript"])
         hvis ikkje _lik(cf_obj["verify_data"], _finished_mac(s["c_hs"], th_sfin)) { returner _feil("client Finished MAC feil") }
         s["transcript"] = _saman(s["transcript"], _slice(cf, 0, ph["next"]))
+        s["cipher"] = "TLS_CHACHA20_POLY1305_SHA256"
+        s["tls_version"] = "TLSv1.3"
         s["fase"] = "etablert"
         s["seq_w"] = 0
         s["seq_r"] = 0
